src/plugins/irc-events/unhandled.js

```diff
@@ -14,6 +15,7 @@ module.exports = function(irc, network) {
 		network.channels[0].pushMessage(client, new Msg({
 			type: Msg.Type.UNHANDLED,
 			command: command.command,
+			commandslug: slugify(command.command),
```

Comment:
Ah you are right. Neither commands, usernames, nicknames, channel names or network names can contain spaces, so there is indeed no need to do that. Probably need to deal with a couple of special characters that are allowed in these though (e.g. dots in hostnames). As I mentioned in the issue, it would be good to slugify all these.